components/list/demo/list-item-sample.js

```diff
@@ -94,6 +95,10 @@ class ListItemSample extends ListItemCheckboxMixin(LitElement) {
 	_handleMouseLeave() {
 		this._hovering = false;
 	}
+
+	_renderDragHandle() {
+		return html`<d2l-list-item-drag-handle ?disabled=${this.disabled}></d2l-list-item-drag-handle>`;
```

Comment:
Nit: While Lit will handle with and without quotes around the attribute value, we've start to opt for trying to use quotes always.  Reason being is that it's closer to the HTML spec.  Minor thing, but will probably come up again in other PRs.

Reply:
Yup, will change that. This could be something a linter code do?